src/neo4j_genai/retrievers/vector.py

```diff
@@ -275,16 +276,17 @@ def _get_search_results(
         """
         try:
             validated_data = VectorCypherSearchModel(
-                vector_index_name=self.index_name,
-                top_k=top_k,
                 query_vector=query_vector,
                 query_text=query_text,
+                top_k=top_k,
                 query_params=query_params,
+                filters=filters,
             )
         except ValidationError as e:
             raise SearchValidationError(e.errors()) from e
 
         parameters = validated_data.model_dump(exclude_none=True)
+        parameters["vector_index_name"] = self.index_name
```

Comment:
I think `filters` can be excluded from these `parameters`, the only use of this variable is  to store the parameters for the Cypher query, right?

Reply:
Yeah it's used to store the params used to format the Cypher query. I'll add a `del` statement to get rid of the filters.